tmk_core/protocol/chibios/chibios.c

```diff
@@ -196,8 +197,6 @@ void protocol_pre_task(void) {
          * the keyboard. Therefore we do nothing here. */
     }
 #endif
-
-    usb_event_queue_task();
```

Comment:
One potential problem with removing `usb_event_queue_task()` from here is that now `USB_EVENT_WAKEUP` won't be processed immediately after the resume.  So maybe we need to do add another `usb_event_queue_task()` in the `USB_SUSPENDED` branch above (instead of the comment about the wakeup callback, which won't actually run until that function is called).

Reply:
Done that, thanks!